src/execSync.js

```diff
@@ -1,7 +1,9 @@
 import child from "child_process";
 
-export default function execSync(cmd) {
-  return child.execSync(cmd, {
-    encoding: "utf8"
-  }).trim();
+export default function execSync(cmd, options = {}) {
+  if (!options.encoding) {
+    options.encoding = "utf8";
```

Comment:
Any reason why to make this optional? When would you not need `utf8` encoding in our case?